### PR TITLE
fixes #13, import fetch API polyfill

### LINKED ISSUE #13: [Library] Import fetch API polyfill
Majaja uses lots of super new Web API - fetch, it doesn't support yet on safari and IE. Thus, we're able to import fetch's polyfill to workaround it.

See also https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#Browser_compatibility

### PR DESCRIPTION
**Pull request by dwi2**
@rickychien Plz help to review it, thanks

### CODE REVIEW

**Comment by dwi2** [comment]
I tested it on Safari and it works! But I haven't test it on IE because I don't have Windows environment.

File: package.json [hunk 1]
```diff
@@ -14,9 +14,12 @@
   },
   "dependencies": {
     "bootstrap": "^3.3.5",
+    "exports-loader": "^0.6.2",
+    "imports-loader": "^0.6.4",
```
Comment (rickychien) [inline package.json]: Both of webpack loaders are dev time libraries, you should put it in devDependencies

**Comment by rickychien** [comment]
GJ dwi2, do you test it on safari?

File: package.json (hunk 1 above)
Comment (dwi2) [inline package.json]: got it

**Comment by dwi2** [comment]
Yeh. It works on Safari.
The only risk on IE is that `whatwg-fetch` use native `promise`. I am not sure if babel handles that for us or not.

**Comment by rickychien** [comment]
Thank you @dwi2 !!!!

**Pull request merged**